### PR TITLE
Don't wrap result items in the new FAR window.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

**Comment by jasonmalinowski** [comment]
:+1:. Tests?

**Comment by CyrusNajmabadi** [comment]
I really don't know a good way to test this.  It'll either be a pointless test (i.e. i call 'wrap: false' in a unit test and i get a texblock with certain properties on it), or it involves writing an entire streaming FAR integration component that pulls apart the entire WPF hierarchy and then sees that the text block has that value.  I didn't think either were worth it.

**Comment by CyrusNajmabadi** [comment]
retest windows_eta_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
retest windows_eta_open_prtest please

File: src/EditorFeatures/Core/Shared/Extensions/TaggedTextExtensions.cs [hunk 1]
```diff
@@ -71,12 +71,17 @@ public static TextBlock ToTextBlock(this TaggedText part, ClassificationTypeMap
...
         {
             classificationFormatMap = classificationFormatMap ?? "tooltip";
             var formatMap = typeMap.ClassificationFormatMapService.GetClassificationFormatMap(classificationFormatMap);
 
-            var textBlock = new TextBlock { TextWrapping = TextWrapping.Wrap };
+            var textBlock = new TextBlock
+            {
+                TextWrapping = wrap ? TextWrapping.Wrap : TextWrapping.NoWrap,
+                TextTrimming = wrap ? TextTrimming.None : TextTrimming.CharacterEllipsis
+            };
```
Comment (DustinCampbell) [inline src/EditorFeatures/Core/Shared/Extensions/TaggedTextExtensions.cs]: line break after? (a la StyleCop)

**Comment by DustinCampbell** [comment]
:+1:

File: src/EditorFeatures/Core/Shared/Extensions/TaggedTextExtensions.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Shared/Extensions/TaggedTextExtensions.cs]: This doesn't warrant it to me as this is just as a normal expression statement with an assignment in it.  If the statement ended with ```}``` i'd feel differently.

**Pull request merged**